content/en/docs/armory-agent/armory-agent-quick.md

```diff
@@ -5,48 +5,146 @@ description: >
   Learn how to install the Armory Agent in your Kubernetes and Armory Enterprise environments.
 weight: 2
 ---
-![Proprietary](/images/proprietary.svg)
+![Proprietary](/images/proprietary.svg) This feature requires an Armory licensed entitlement.
 
-## Compatibility matrix
+## Overview
 
-{{< include "agent/agent-compat-matrix.md" >}}
-
-The Agent consists of a service deployed as a Kubernetes `Deployment` and a plugin to Spinnaker's Clouddriver service. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
+The Agent consists of a plugin to Spinnaker's Clouddriver service and a K8s deployment that connects to Clouddriver. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
 
 ## {{% heading "prereq" %}}
 
-* This guide is for experienced Kubernetes and Armory Enterprise users.
+* This guide assumes you're using the Armory Operator to install Spinnaker, with the Kustomize method from the [spinnaker-kustomize-patch repo](https://github.com/armory/spinnaker-kustomize-patches).
```

Comment:
The `spinnaker-kustomize-patches` repo above is a complete set of patches, the expectation is not to use it as is, but to grab example patches as needed. I would not make it a requirement as it's a no go for most users.

I'd say:
- either you use kustomize and just include that patch - it's easy to configure the plugin.
- or you don't use kustomize in which case you'll need to insert the following configuration blocks in `SpinnakerService`.

Reply:
We want our users to use this repo and method. trying to get this to a quick start that doesn't go off on multiple angles to explain this way vs that way.

Reply:
@ncknt We understand the purpose is to give people options, at the same time we want to promote opinionated configs.  It may mean reworking some of recipes or to move some of the configs to `expert` configuration to keep people from shooting themselves in the foot while they are trying to learn the concepts.